Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -546,6 +546,12 @@ extension _ProtocolClient : URLProtocolClient {
         guard let task = `protocol`.task else { fatalError() }
         guard let session = task.session as? URLSession else { fatalError() }
         switch session.behaviour(for: task) {
+        case .taskDelegate(let delegate) where delegate as? URLSessionDownloadDelegate != nil:
```

Comment:
I think this can be `case .taskDelegate(let delegate) where delegate is URLSessionDownloadDelegate:`